release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1300,6 +1300,12 @@ The Amazon Web Services (AWS) Elastic File System (EFS) Technology Preview featu
 
 * Technology preview badges were displayed on the OpenShift Pipelines workflows in the web console, though OpenShift Pipelines was GA. The Technology preview badges are now removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1945153[*BZ#1945153*])
 
+* Previously, pipelines created in the Git import flow for private repositories failed to run. This happened because the pipeline `ServiceAccount` did not use secrets created by the Git import flow for private Git repositories. With this update, you can add a secret name to the annotations of the pipeline `ServiceAccount` object, and add pipeline-specific annotations to the provided secret. As a result, pipelines for private Git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970485[BZ#1970485])
+
+* Previously, the Start Modal field did not recognize an empty string as a valid entry, so you had to enter a value. This happened even though the Tekton Operator recognizes empty strings as a valid `parameter default`. With this update, you have the option to enter an empty string. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966275[BZ#1966275])
+
+* Previously, a decorator was shown for revisions in *Topology view* because annotations were passed to specifications of Knative service along with metadata. Now the required decorators are shown for the knative service only in the *Topology view* and not with the associated revisions. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954962[BZ#1954962])
```

Comment:
I think passive voice is fine here, especially considering that we want the RN in today....however, the problem with passive voice is that it often leaves questions unanswered to users. 

For instance, reading, `annotations were passed to specifications of Knative service`, I don't know who/what passed the annotations to the specifications. 

I can't change this sentence to active voice, because I don't know who/what is doing the action. Here are some suggestions for how to make the rest of the bullet point active, if you wish (again, I don't think passive voice reads badly here, so I think it's fine to be kept as is):

`Previously, the *Topology* view showed a decorator for revisions because the annotations were passed to the specifications of Knative service along with the metadata. Now,  only the *Topology* view shows the required decorators for the knative service.`

Reply:
K capital in Knative in the last sentence.

Reply:
Please use the same wording as here: https://github.com/openshift/openshift-docs/pull/34411/files#diff-d1bde4944d8eb8c20936e6de118951cc975445318afef3c7d559ef4375ebb729R1572 This part: "The current release fixes this issue by passing annotations only to the Knative service metadata." is important and helps user understand the issue better.

Reply:
Done